core/class.go

```diff
@@ -50,10 +52,16 @@ func (c *Cairo0Class) Hash() (*felt.Felt, error) {
 	return cairo0ClassHash(c)
 }
 
+func (c *Cairo0Class) SierraVersion() string {
+	return "0.1.0"
```

Comment:
Which source we have on this value (cc: @rianhughes). Looking at the Blockifier code, deprecated Sierra is 0.0.0 👀. It actually doesn't affect the blockifier execution, because literally everything below 2.9.0 is executed with the VM.

Reply:
I see what you mean. Fixing it now